DNN Platform/Website/admin/Skins/BreadCrumb.ascx.cs

```diff
@@ -172,21 +172,19 @@ protected override void OnLoad(EventArgs e)
                     tabUrl = this._navigationManager.NavigateURL(tab.TabID, string.Empty, "GroupId=" + this.GroupId);
                 }
 
-                // Begin breadcrumb
-                this._breadcrumb.Append("<span itemprop=\"itemListElement\" itemscope itemtype=\"http://schema.org/ListItem\">");
-
-                // Is this tab disabled? If so, only render the text
-                if (tab.DisableLink)
-                {
-                    this._breadcrumb.Append("<span class=\"" + this._cssClass + "\" itemprop=\"name\">" + tabName + "</span>");
-                }
-                else
-                {
-                    this._breadcrumb.Append("<a href=\"" + tabUrl + "\" class=\"" + this._cssClass + "\" itemprop=\"item\"><span itemprop=\"name\">" + tabName + "</span></a>");
+                // Is this tab disabled? If so, only render a span
+                if (tab.DisableLink)
+                {
+                    this._breadcrumb.Append("<span class=\"" + this._cssClass + "\">" + tabName + "</span>");

```

Comment:
I like this change; it makes sense and makes things better. One potential issue I see is that we're removing a `span` that wrapped the disabled page, which may have been used for styling. Should we continue to wrap this `span` in another `span`? We can't do anything if that `span` was targeted using the metadata attributes, but it seems likely to me that there are themes which expect that wrapping element.

Thoughts @dnnsoftware/approvers?

Reply:
I noticed as well that a nested span without an id or class has been removed - IMO the effect on skins should be minimal and rendered HTML is even clearer without.

Reply:
I agree with your comment @bdukes, some themes will likely need to be restyled (though minimal effort probably as @sleupold points out)-- I've seen several themes including our own that target with that wrapping `<span>`

Reply:
Thank you for your feedback. I could try later today to wrap the span inside of an additional span. That would probably provide better compatibility.

Reply:
I just tested to put another span around the span and it looks the same visually. Shall I change the code accordingly?

Reply:
I think at this point it's probably more likely to cause a problem if it's missing than to cause a problem by being added. Unless someone has a different opinion, please go ahead and add it in. Thanks!